gulpfile.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var metal = require('gulp-metal');
+
+metal.registerTasks({
+	bundleCssFileName: 'autocompleteBadges.css',
+	bundleFileName: 'autocompleteBadges.js',
+	moduleName: 'metal-autocompleteBadges',
+	testNodeSrc: [
```

Comment:
This option is only necessary for metal modules that need to run on Node JS, making it easier to run the tests there as well. UI components don't need to run on node though (since it doesn't even have a dom), just on browsers, so can you remove this option (and the entire `env` directory as well)? They were probably added automatically by `generator-metal` after receiving a `yes` to the question about node environment :)